modules/packages/UnitTest.chpl

```diff
@@ -612,7 +610,7 @@ module UnitTest {
       seq2: The second sequence to compare.
       seq_type_name: The name of datatype of the sequences
     */
-    proc assertSequenceGreater(seq1, seq2, seq_type_name) throws {
+    proc assertSequenceGreater(seq1, seq2, param seq_type_name) throws {
```

Comment:
What's the motivation for adding `param` here?

I don't know about the UnitTest module that much, but this sounds like it'd need a design discussion first.

Reply:
That's described in the OP, where the challenge was (a) we can't keep using array.indices because of the warning (and since this is a module rather than test code, we can't rely on the param), and (b) we can't apply `.domain` to types other than arrays that are passed into the routine (so compilation broke if this wasn't a param).  All that said, note that this is an internal routine (it's no doc'd) and that it's always called with a param (which should really be an enum, but...).

Reply:
Ah, I didn't see `no doc` as it was before the header comment.

(Sorry for not checking the PR message.)